### PR TITLE
uploading JavaScript Assignment 2,3,4

### PR DESCRIPTION
**Pull request by vishwaraj1706**

### CODE REVIEW

**Review by harshalkotkar — APPROVED** [review]

File: vishwa@work/JavaScript/Assignment_4/krishnamurthy.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        }
+    </style>
+  </head>
+  <body>
+      <label>Number : </label>
+      <input type="text" id="num"><br><br>
+      <button type="submit" onclick="krishna()">Krishnamurthy Number</button><br><br>
+      <p id="murthy"></p>
+  </body>
+</html>
```
Comment (Pawan1005) [inline vishwa@work/JavaScript/Assignment_4/krishnamurthy.html:59]: check output  of this program

File: vishwa@work/JavaScript/Assignment_2/percentage.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    Mathematics : <input type="text" id="math"><br><br>
+    Science : <input type="text" id="sci"><br><br>
+    Social Science : <input type="text" id="ss"><br><br>
+    <p id="tp"></p>
+    <p id="total"></p>
+    <p id="grd"></p>
+    <button type="button" onclick="calc()">Calculate</button>
+    <button type="button" onclick="grade()">Grade</button>
+  </body>
+</html>
```
Comment (Pawan1005) [inline vishwa@work/JavaScript/Assignment_2/percentage.html:63]: indentation is wrong

File: vishwa@work/JavaScript/Assignment_2/secondlargestnumber.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        Number 2: <input type="text" id="num2"><br><br>
+        Number 3: <input type="text" id="num3"><br><br>
+        Number 4: <input type="text" id="num4"><br><br>
+        Number 5: <input type="text" id="num5"><br><br>
+        <button type="submit" onclick="find()">Second Largest Number</button>
+    </form>
+    <h2>Second Large Number from List</h2>
+    <p id="secondnumber"></p>
+  </body>
+</html>
```
Comment (Pawan1005) [inline vishwa@work/JavaScript/Assignment_2/secondlargestnumber.html:40]: same indentation mistake

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
indentation is wrong in files  make it correct

**Pull request closed without merging**